components/fxa-client/examples/oauth_flow.rs

```diff
@@ -19,6 +19,8 @@ fn main() {
     let code = &query_params["code"];
     let state = &query_params["state"];
     fxa.complete_oauth_flow(&code, &state).unwrap();
+    let auth_status = fxa.check_authorization_status(SCOPES[0]);
+    println!("auth_status: {:?}", auth_status);
```

Comment:
@rfk What about this one? This leads to is hitting the introspection_endpoint formatted as `introspection_endpoint: format!("{}/v1/introspect", resp.auth_server_base_url),`. Is this an up and running endpoint?